experimental/datacommons-js/README.md

```diff
@@ -0,0 +1,366 @@
+
+Draws bar chart tile with specified [statistical variable](https://datacommons.org/tools/statvar).
+
+- `element` (HTMLElement)
+
+  DOM element to attach the chart
+
+- `props` (BarTilePropType)
+
+  Chart configuration object
+
+  - `id` (string)
+
+    DOM id attribute to attach to the chart
+
+  - `title` (string)
+
+    Chart title
+
+  - `place` (NamedTypedPlace)
+
+    Scope the statistical variable query by this place configuration
+
+    - `dcid` (string): Place DCID
+    - `name` (string): Place name
+    - `types` (string[]): [Place types](https://datacommons.org/browser/Place)
+
+  - `enclosedPlaceType` (string)
+
+    Enclosing place type
+
+  - `comparisonPlaces` (string[])
+
+    A list of related places dcids to show comparison with the main place.
+
+  - `statVarSpec` (StatVarSpec)
+
+    Datacommons [statistical variable](https://datacommons.org/tools/statvar) configuration
+
+    - `statVar` (string): Statistical variable dcid
+    - `denom` (string): Denominator (e.g., `Count_Person`)
+    - `unit` (string): [Unit](https://datacommons.org/browser/UnitOfMeasure) (e.g., '$')
+    - `scaling` (number):
+    - `log` (boolean): Set to true to color values using logarithmic scale
+    - `name?` (string):
+
+  - `svgChartHeight` (number)
+
+    Height (px) for the underlying SVG chart.
+
+  - `className` (string)
+
+    Extra classes to add to the container.
+
+  - `isDataTile?` (boolean)
+
+    Whether or not to render the data version of this tile
+
+  - `apiRoot` (string)
+
+    Datacommons API root. Default: `https://datacommons.org`
+
+### `datacommons.drawLine`
+
+Draws line chart tile with specified [statistical variable](https://datacommons.org/tools/statvar).
+
+- `element` (HTMLElement)
+
+  DOM element to attach the chart
+
+- `props` (LineTilePropType)
+
+  Chart configuration object
+
+  - `id` (string)
+
+    DOM id attribute to attach to the chart
+
+  - `title` (string)
+
+    Chart title
+
+  - `place` (NamedTypedPlace)
+
+    Scope the statistical variable query by this place configuration
+
+    - `dcid` (string): Place DCID
+    - `name` (string): Place name
+    - `types` (string[]): [Place types](https://datacommons.org/browser/Place)
+
+  - `comparisonPlaces` (string[])
+
+    A list of related places dcids to show comparison with the main place.
+
+  - `statVarSpec` (StatVarSpec)
+
+    Datacommons [statistical variable](https://datacommons.org/tools/statvar) configuration
+
+    - `statVar` (string): Statistical variable dcid
+    - `denom` (string): Denominator (e.g., `Total_Population`)
+    - `unit` (string): [Unit](https://datacommons.org/browser/UnitOfMeasure) (e.g., '$')
+    - `scaling` (number):
+    - `log` (boolean): Set to true to color values using logarithmic scale
+    - `name?` (string):
+
+  - `svgChartHeight` (number)
+
+    Height (px) for the underlying SVG chart.
+
+  - `className` (string)
+
+    Extra classes to add to the container.
+
+  - `isDataTile?` (boolean)
```

Comment:
same here, skip these two params for now?

Reply:
done